test/runtests.jl

```diff
@@ -11,7 +11,8 @@ testfiles = (
     "intervals.jl",
     "broadcasting.jl",
     "identities_Euler.jl",
-    "fateman40.jl"
+    "fateman40.jl",
+    "staticarrays.jl"
```

Comment:
Currently, Aqua tests are run twice; profitting this PR, could you remove the "aqua.jl" entry (in line 6 of this file)?

Reply:
It's done now 👍